src/paths/all_shortest_paths.c

```diff
@@ -174,10 +181,20 @@ int igraph_get_all_shortest_paths(const igraph_t *graph,
     }
 
     /* from -> from */
-    vptr = igraph_Calloc(1, igraph_vector_t); /* TODO: dirty */
+    vptr = igraph_Calloc(1, igraph_vector_t);
+    if (vptr == 0) {
+    IGRAPH_ERROR("cannot run igraph_get_all_shortest_paths", IGRAPH_ENOMEM);
+    }
     IGRAPH_CHECK(igraph_vector_ptr_push_back(&paths, vptr));
     IGRAPH_CHECK(igraph_vector_init(vptr, 1));
     VECTOR(*vptr)[0] = from;
+    vptr_e = igraph_Calloc(1, igraph_vector_t);
+    if (vptr_e == 0) {
+    IGRAPH_ERROR("cannot run igraph_get_all_shortest_paths", IGRAPH_ENOMEM);
```

Comment:
Please fix the indentation here; on my screen it is indented at the same level where the `if` is.